locales/en/LC_MESSAGES/volto.po

```diff
@@ -2992,6 +2992,11 @@ msgstr "in"
 msgid "rrule_nd"
 msgstr "nd"
 
+#: overrideTranslations
+# defaultMessage: di
+msgid "rrule_on"
+msgstr ""
```

Comment:
Metti anche questa dato che le altre ci sono per la ricorrenza. Sono stringhe che il pubblico vede quindi i siti multilingua devono mostrarle in inglese